pkg/commands/resolver.go

```diff
@@ -224,9 +224,16 @@ func makePublisher(po *options.PublishOptions) (publish.Interface, error) {
 		// If not publishing, at least generate a digest to simulate
 		// publishing.
 		if len(publishers) == 0 {
+			// If one or more tags are specified, use the first tag in the list
+			tag := "latest"
```

Comment:
This seems like a behavior change?

Reply:
`po.Tags` is defaulted to `[]string{"latest"}` if not set, so this isn't exactly a behavior change, but it does subtly depend on logic elsewhere which might change and break assumptions.

Fixed!